changelog.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 0.14.0-rc1
+
+* Add support for selecting and deleting multiple sources (#2208, 2188, #2230, #2252)
```

Comment:
For the release announcement itself we should go into more detail about this, like the top pane moving to the bottom, etc.